packages/management/core-management-plugins/bundle/config/feature.yaml

```diff
@@ -0,0 +1,29 @@
+#@ load("@ytt:data", "data")
```

Comment:
The [other files](https://github.com/vmware-tanzu/tanzu-framework/tree/main/packages/management/core-management-plugins/bundle/config) in this directory have a numerical prefix (probably ordering). Should this also follow the same convention?

Reply:
numerical prefix doesn't change the order of creation of resources in the package, ordering is done using ordering annotations https://carvel.dev/kapp/docs/v0.45.0/apply-ordering/, ill make another commit to this pr to remove the prefix of the other files